src/HttpExponentialBackoff.js

```diff
@@ -139,7 +139,7 @@ class HttpExponentialBackoff {
    */
   __getRetryDelayWithRetryAfterHeader (initialDelayInMillis) {
     return (attempt, error, response) => {
-      const retryAfter = response.headers.get('Retry-After')
+      const retryAfter = response?.headers.get('Retry-After') || null
```

Comment:
not to nitpick too much but if `response` is null, the optional chaining here would return `undefined` right? Wondering if `|| null` is needed